.github/ISSUE_TEMPLATE.md/feature_request.md

```diff
@@ -0,0 +1 @@
+# # # Ready to be writtenn
```

Comment:
I prefer to not include a empty file, can you remove this file and the bug report?